library/pkparse.c

```diff
@@ -219,6 +219,7 @@ static int pk_get_ecparams(unsigned char **p, const unsigned char *end,
     return 0;
 }
 
+#if defined(MBEDTLS_ECP_LIGHT)
```

Comment:
Note: again, #7442 will change that (again, just for reference, nothing to change here).